client/settings/digital-wallets/test/index.test.js

```diff
@@ -22,6 +71,8 @@ describe( 'DigitalWallets', () => {
 			cartCheckbox,
 		] = screen.getAllByRole( 'checkbox' );
 
+		userEvent.click( screen.getByText( 'Enable 1-click checkouts' ) );
```

Comment:
The title of the test is "should disable 1-click checkout sections if 1-click checkout is disabled". But this is _enabling_ the 1-click checkout checkbox - is that accurate?

Reply:
Yes, the titles for this and the next test cases made no sense. Seems like I misplaced both. The test cases were meant to test disable->enable and enable->disable. I will update these. Thanks.

Reply:
Updating to "should enable 1-click checkout sections if 1-click checkout is enabled". The action of checking the button from unchecked->check is to enable.